librz/util/sdb/src/buffer.h

```diff
@@ -0,0 +1,64 @@
+#ifndef BUFFER_H
```

Comment:
same here. RzBuffer ?

Reply:
it is very different for now... if it won't be necessary anymore later we can remove it, it's not a lot of code.